build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```

Comment:
I read your comment in previous PR. Are you using Android Studio 2.1.3? This plugin is compatible with 2.2.0 maybe that is the cause for the error.